src/opcodes/opcodes.cairo

```diff
@@ -261,6 +269,26 @@ pub mod Opcode {
             163 => arithmetic::opcode_min(ref engine),
             164 => arithmetic::opcode_max(ref engine),
             165 => arithmetic::opcode_within(ref engine),
+            166 => utils::not_implemented(ref engine),
+            167 => utils::not_implemented(ref engine),
+            168 => utils::not_implemented(ref engine),
+            169 => utils::not_implemented(ref engine),
+            170 => utils::not_implemented(ref engine),
+            171 => utils::not_implemented(ref engine),
+            172 => utils::not_implemented(ref engine),
+            173 => utils::not_implemented(ref engine),
+            174 => utils::not_implemented(ref engine),
+            175 => utils::not_implemented(ref engine),
+            176 => utils::opcode_reserved("NOP1", ref engine),
```

Comment:
NOP opcodes are no operation, not reserved opcodes. You will want to use opcode_nop https://github.com/keep-starknet-strange/shinigami/blob/53ab9c10d70df7667ebab14903c19dabd325d367/src/opcodes/flow.cairo#L17